server/config.go

```diff
@@ -37,6 +38,8 @@ type Config struct {
 	DataDir     string
 	RestoreFile *string
 
+	LeveldbOptions *leveldb.Options
```

Comment:
Would be a little confused in this option.

Should we use basic types to wrapped it?

Reply:
Not recommended, more parameters may be added here and using basic types will make modifications cumbersome

Reply:
OK, you convinced me.